OpenSim/Tests/VisualizeModel/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(VISUALIZE_MODEL visualizeModel)
 
 add_executable(${VISUALIZE_MODEL} VisualizeModel.cpp)
 
-target_link_libraries(${VISUALIZE_MODEL} osimTools)
+target_link_libraries(${VISUALIZE_MODEL} osimAnalyses)
```

Comment:
Not sure why osimAnalyses here, in the past I used osimTools which is the top of stack due to using OpenSim.h, now that we're using dev. environment headers we likely can get away with using osimActuators (to pull in muscles) but I don't know if you tried that. Glad you found this utility useful.

Reply:
What would happen if you tried to visualize a Model with Analyses?

Reply:
Analyses get ignored similar to any other plugins. The goal is to minimize dependencies if possible.

Reply:
I did not try depending on only `osimActuators`, but I thought it wouldn't work for the reason @aseth1 mentioned. I agree with minimizing dependencies and speeding up compile time but this is separate from the purpose of the PR and I would prefer to not try to further optimize here.